.github/workflows/pr-docker-check.yml

```
@@ -26,25 +26,28 @@ jobs:
       - name: Lint Dockerfile
         run: bun scripts/lint-docker-file
 
+      # Log in to GitHub Container Registry (GHCR)
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)